eng/testing/performance/performance-setup.sh

```diff
@@ -358,6 +364,10 @@ if [[ "$physicalpromotion" == "true" ]]; then
     configurations="$configurations PhysicalPromotionType=physicalpromotion"
 fi
 
+if [[ "${hybridglobalization,,}" == "true" ]]; then # convert to lowercase to test
+    configurations="$configurations HybridGlobalization=True" # Force True for consistency
+fi
+
```

Comment:
super nit: ```suggestion
if [[ "${hybridglobalization,,}" == "true" ]]; then # convert to lowercase to test
    configurations="$configurations HybridGlobalization=True" # Force True for consistency
fi
```

Reply:
I don't see a difference with the suggestion, is there something I am missing for this?

Reply:
No difference in the patch there. And I'm not sure what I had in mind now! Looks fine :)